pupil-spa/angular.json

```diff
@@ -11,7 +11,6 @@
         "build": {
           "builder": "@angular-devkit/build-angular:browser",
           "options": {
-            "aot": true,
```

Comment:
is this now the `buildOptimizer` flag below?

Reply:
I don't think so.  The angular migration script performed this deletion, so I guess it is no longer supported.